test/fixtures/components/container-with-multiple-nested/_package/server.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Not atm (there is a PR and good progress though toward that -> https://github.com/okonet/lint-staged/pull/188). For the time being I could just make sure we test against `test/acceptance` && `test/unit`